HeimdallrTests/Core/HeimdallrSpec.swift

```diff
@@ -725,6 +725,34 @@ class HeimdallrSpec: QuickSpec {
                         expect(result?.error?.code).to(equal(OAuthErrorInvalidClient))
                     }
                 }
+
+                context("when issueing multiple requests at the same time") {
+                    it("only issues one refresh token call") {
```

Comment:
Please note that this test does not verify the sequential execution of requests. What you actually want to verify is that the second request isn't started before the first completion block got called.